packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -4,139 +4,105 @@ excerpt: >-
+![Access the dashboard by email notification](/img/RLDashboard1.png "Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard")
 
-* Bob: (IP1 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
-* Alice: (IP2 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* You've hit a rate limit when you're in the rate limits dashboard or the Admin console, and a banner displays to inform you. Click **utilization history** in the banner to open the Rate Limits Dashboard for that particular endpoint or **syslog** to open the System Log.
 
-Let's assume the org-wide quota for the `/authorize` endpoint is a total of 2,000 requests per minute on Custom Authorization Servers. Bob decides to run a batch job that triggers about 2,000 `/authorize` requests per minute.
+![Access the dashboard by link in the banner](/img/RLDashboard2.png "Displays the banner in the dashboard that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
 
-Without the client-based rate limit framework, Bob consumes all of the total allowed requests per minute (2,000 requests per minute). This results in HTTP 429 errors for both Bob and Alice, which makes the application inaccessible for everyone.
+* You've hit a rate limit warning or violation, and a rate limit event (violation or warning) is created in the System Log from where you can click **View Utilization** under **Event info** to go to the rate limits dashboard. All the rate limit events in the System Log (violations and warnings) have a link to the rate limit dashboard.
 
-With the client-based rate limit framework enabled, after Bob exceeds his individual limit of 60 requests per minute, HTTP 429 errors are sent. Requests that originate from Bob's IP address, and the specific device from which the requests are made, start receiving the errors. Alice continues to access the application without any issues.
+![Access the dashboard by link in the System Log](/img/RLDashboard3.png "Displays the event in the System Log that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
 
-![Client-based isolation for a unique IP address](/img/diagram1.png "Displays Bob as portal123 IP1 and Alice as portal123 IP2, both making authorize requests")
+## Key components of the dashboard
 
-#### Client-based isolation for users accessing `/authorize` from a NAT IP
+The rate limits dashboard provides utilization for the endpoint based on a set of statistics and shows multiple views of the data utilization.
```

Comment:
```suggestion
The rate limits dashboard provides use for the endpoint based on a set of statistics and shows multiple views of the data use.
```

Does the rate limits dashboard provide use for the endpoint or data on endpoint use?